android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/DatabaseTest.java

```diff
@@ -321,6 +321,7 @@ public void testGetExistingDocWithIDFromDifferentDBInstance() throws CouchbaseLi
     @Test
     public void testGetExistingDocWithIDInBatch() throws CouchbaseLiteException {
         final int NUM_DOCS = 10;
+        Database.getLog().getConsole().setLevel(LogLevel.VERBOSE);
```

Comment:
Is this for testing if the console log works? Can we delete it if we don't need it?